helm_deploy/prison-visits-2/values.yaml

```diff
@@ -21,7 +21,7 @@ generic-service:
 
   ingress:
     enabled: true
-    tlsSecretName: prison-visits-booking-cert
+    tlsSecretName: prison-visits-booking-staff-cert
```

Comment:
change this to `prison-visits-staff-justice-cert`

Reply:
prod/dev have different certificate names - would it be simpler to rectify this in CP environments to all be the same cert across environments?

And double check the host listed in that cert